i18n/sv-SE/cosmic_term.ftl

```diff
@@ -0,0 +1,67 @@
+# Context sidor
+
+## Settings
+settings = Inställningar
+
+### Appearance
+appearance = Utseende
+theme = Tema
+match-desktop = Matcha skrivbordet
+dark = Mörkt
+light = Ljust
+syntax-dark = Färgschema mörkt
+syntax-light = Färgschema ljust
+default-zoom-step = Zoom steg
+
+### Teckensnitt
+font = Teckensnitt
+advanced-font-settings = Avancerade teckensnittsinställningar
+default-font = Teckensnitt
+default-font-size = Teckenstorlek
+default-font-stretch = Teckenstretch
+default-font-weight = Normal teckensnittsvikt
+default-dim-font-weight = Dämpad teckensnittsvikt
+default-bold-font-weight = Fet teckensnittsvikt
+use-bright-bold = Gör fet text ljusare
+
+### Delar
+splits = Delar
+focus-follow-mouse = Skrivfokus följer mus
+
+### Avancerat
+advanced = Avancerat
+show-headerbar = Visa rubrikrad
+show-header-description = Visa rubrikrad från högerklicksmenyn.
+
+# Sök
+find-placeholder = Sök…
```

Comment:
This should either be ... with three dots like Inställningar... or the other way around.